main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -70,6 +70,11 @@ class Flix {
     "Div.flix" -> LocalResource.get("/src/library/Div.flix"),
     "Rem.flix" -> LocalResource.get("/src/library/Rem.flix"),
     "Exp.flix" -> LocalResource.get("/src/library/Exp.flix"),
+    "And.flix" -> LocalResource.get("/src/library/And.flix"),
```

Comment:
Wait, why are the old names back?

Reply:
God dammit. Unattentive rebase in IntelliJ i think. I'll fix it.